package.json

```diff
@@ -4,7 +4,8 @@
   "description": "work by HAOCHEN ZOE ZHENGNAN",
   "main": "index.js",
   "scripts": {
-    "start": "node server.js",
+    "start": "npm run build",
+    "build": "webpack && node server.js",
```

Comment:
i think you are a bit confused.
start: should start the node with the server.js, thats all. In your case you call the build. If you want of course you can call the build from the start, but it is not crucial.
build: should create the bundle.js but it should not start the server, it just builds up the system.